### PR TITLE
Added ability to bundle arbitrary files in deployment zip.

### PR DESCRIPTION
**Pull request by wunderk1nd-e**
Removed the setting/code that bundled *.config files in the ebextensions directory into the deployment zip.
Added a setting that allows for arbitrary directories/files to be added to a directory named eb which will be included in the deployment bundle.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -31,4 +31,5 @@ SBT Elastic Beanstalk Plugin exposes the following settings:
 | awsRegion             | Region for the elastic beanstalk application and S3 bucket      | *mandatory*                   |
 | awsVersion            | Version number to tag release with in elastic beanstalk         | defaults to the __version__   |
 | awsDockerAuthLocation | Name of the S3 bucket + key containing a docker auth config     | defaults to None              |
-| awsEbextensionsDir    | Directory containing \*.config files for advanced customisation | defaults to __ebextensions__  |
+| awsBeanstalkAdditionalFilesDir    | Directory containing files/directories _(such as `.ebextensions`)_ for advanced customisation | defaults to __eb__  |
```
Comment (yesdaveOvo) [inline README.md]: might want to note here that this used to be `awsEbextensionsDir` so anyone upgrading versions has a better idea what to do

Comment (wunderk1nd-e) [reply README.md]: I've put the old column back in with a strikethrough and migration description

File: build.sbt [hunk 2]
```diff
@@ -1,7 +1,7 @@
 name := "sbt-elastic-beanstalk-plugin"
 organization := "com.ovoenergy"
 organizationName := "OVO Energy"
-version := "3.0.1"
+version := "3.1.1"
```
Comment (cb372) [inline build.sbt]: 4.0.0 as it's a breaking change?

Comment (wunderk1nd-e) [reply build.sbt]: Sounds good

**Pull request merged**